models/rankingSchema.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const rankingSchema = new Schema(
+  {
+    _id: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: [true, "ID do produto é obrigatório"],
+    },
+    name: {
+      type: String,
+      required: [true, "Nome é obrigatório"],
+    },
+    date: {
+      type: Date,
+      required: [true, "Data é obrigatória"],
+    },
+    score: {
+      type: String,
+      required: [true, "Score é obrigatório"],
+    },
+  },
+  { collection: "game" }
```

Comment:
Usually  collection names are in plural, check if you can change this later to "games"